RELEASE_NOTES

```diff
@@ -1,5 +1,5 @@
 ========================================================================
-Amazon CloudWatch Agent 1.247358.0 (2023-01-30)
+Amazon CloudWatch Agent 1.247358.0b252413 (2023-01-30)
```

Comment:
What would be the demand for this though? For easier installation of the previous agent, if that's the case, should we have a force mechanism to always install with `1.247358.0` (version without hash) instead of always having `1.247358.0b252413` (hash). 

Small nitpick would be update the hash from the [release Note too](https://github.com/aws/amazon-cloudwatch-agent/releases/tag/v1.247358.0) if the change are merged.

Reply:
As it will change the tag and tagging happens before the hashing, I will edit the description of the release but not the tag.